controllers/vcluster_controller.go

```diff
@@ -166,22 +166,28 @@ func (r *VClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request) (_
 	// check if we have to redeploy
 	err = r.redeployIfNeeded(ctx, vCluster)
 	if err != nil {
-		r.Log.Infof("error during virtual cluster deploy %s/%s: %v", vCluster.Namespace, vCluster.Name, err)
+		r.Log.Info("error during virtual cluster deploy",
+			"namespace", vCluster.Namespace,
+			"clusterName", vCluster.Name,
+			"err", err)
```

Comment:
Technically it's an error, so we probably should use an error logger here:

```suggestion
		r.Log.Error(err, "error during virtual cluster deploy",
			"namespace", vCluster.Namespace,
			"name", vCluster.Name,
		)
```